CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 ## CHANGELOG
 
-### 3.X.X (2021-XX-XX)
+### 3.5.0 (2021-07-26)
```

Comment:
You also need to add the fix about File Input `accept` types: https://github.com/rakutentech/android-miniapp/pull/332

Reply:
Yeah, updated.